tests/integration/slowquery/compatibility_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
```

Comment:
since there are mock and real dbs, how about use file name `compatibility_test.go`.